components/theme/colors.ts

```diff
@@ -77,6 +77,12 @@ const accent3 = {
   accent3: '#d9de3a',
 };
 
+const green_accent = {
```

Comment:
Added these colours for a "non-error" theme on the simple notification, but expect will be unified in the theming task